README.md

```diff
@@ -131,6 +131,36 @@ function join() { return Array.prototype.slice.call(arguments).join(''); }
 curry(join, 3)(1)(0)(1); // "101"
 ```
 
+## del
+
+Functional version of delete obj[key] which returns the same obj without the deleted key.
```

Comment:
Ill add it for you :)


—
@tjmehta

On Mon, Mar 2, 2015 at 6:55 PM, Christoph Hermann
<notifications@github.com> wrote:

>> @@ -131,6 +131,36 @@ function join() { return Array.prototype.slice.call(arguments).join(''); }
>>  curry(join, 3)(1)(0)(1); // "101"
>>  ```
>>  
>> +## del
>> +
>> +Functional version of delete obj[key] which returns the same obj without the deleted key.
> Will add the note as soon as I'm on a labtop. I'm in military this month (mandatory in Switzerland) so it's kind of hard :smile: .
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tjmehta/101/pull/67/files#r25659860